imp_wrappers/pettingzoo_struct.py

```diff
@@ -0,0 +1,58 @@
+from imp_wrappers.pettingzoo_env import ParallelEnv

```

Comment:
I think it is better to have from pettingzoo import Parallel Env and people to install petting zoo in their python environment.

Reply:
Agreed and template removed :)